versions.tf

```
@@ -1,4 +1,7 @@
 
 terraform {
   required_version = ">= 0.12"
+  required_providers {
+    google = "~> 2.19.0"
```

We should pin required providers at the minor level to ensure backward compatible updates can be pulled in without a module release.

```suggestion
    google = "~> 2.19"
```